nixos/modules/services/hardware/udisks2.nix

```
@@ -21,6 +22,17 @@ in
 
       enable = mkEnableOption (lib.mdDoc "udisks2, a DBus service that allows applications to query and manipulate storage devices");
 
+      mountAtMedia = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc ''
```

```suggestion
        description = mdDoc ''
```

Why such an insanity people use  `lib.mdDoc`  when they already opened the can of worms from `with lib`?

---

It is a victim of copy paste.